src/models/GlobalStyleSheet.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+/* Wraps glamor's stylesheet and exports a singleton for global styles. */
+import { StyleSheet } from '../vendor/glamor/sheet'
+
+/* Don't specify a maxLength, since these rules are defined at initialization
+*  and should remain static after that */
+export default new StyleSheet({ speedy: false })
```

Comment:
I don't think we want to do that, because `speedy: true` apparently breaks the chrome dev tools.  

https://github.com/threepointone/glamor/blob/master/src/sheet.js#L70 

So if we do enable it, it should probably be an optional "production only" thing.